<launch>
    <node name="record_path" pkg="path_smoothing_ros" type="record_path" output="screen"/>
        <param name="minimum_distance" value="3"/>
        <param name="path_file" value="$(find path_smoothing_ros)/paths/path.csv"/>

    <node name="path_to_goals" pkg="path_smoothing_ros" type="path_to_goals" output="screen">
        <remap from="/path" to="/path"/>
        <remap from="/waypoint_pose" to="/initialpose"/>

    </node>
    
</launch>